### DISCUSSION TITLE
[SANDBOX] Mode Marketplace sous-boutique type particulier

### DISCUSSION THREAD

**Discussion by s4mf3nn** [PayGreen/documentation#23] [Q&A] [answered]
Bonjour,

Dans le cadre d'une création d'une marketplace C2C, je souhaite créer une sous-boutique. Pour cela j'effectue une requête au endpoint POST /account/shops. Le serveur me retourne un objet avec un type 'COMPANY'. Existe t-il un type 'INDIVIDUAL' ou équivalent pour la création d'un compte particulier ?

Pour info, j'ai basculé la boutique principale en mode 'marketplace' avec la valeur 'economic_model' à 'C2C'.

**Comment by gaelDolle** [comment, marked as answer]
Bonjour,

Merci de vous rapprocher du service client afin de vérifier votre configuration : [Service client](https://paygreen.atlassian.net/servicedesk/customer/portal/9)

Cordialement
